From 56ac0feca7bf1975b2744eb8bff42ac75b7ddb17 Mon Sep 17 00:00:00 2001
From: tinnuadan <tinnuadan@gmail.com>
Date: Fri, 21 Aug 2020 10:17:23 +0200
Subject: [PATCH] inserting blacklisting into build.sh

---
 scripts/build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/build.sh b/scripts/build.sh
index 9b8fa286a7..4b30f12ae3 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -5,7 +5,7 @@ set -e
 basedir="$(cd "$1" && pwd -P)"
 gitcmd="git -c commit.gpgsign=false"
 
-($gitcmd submodule update --init && ./scripts/remap.sh "$basedir" && ./scripts/decompile.sh "$basedir" && ./scripts/init.sh "$basedir" && ./scripts/applyPatches.sh "$basedir") || (
+($gitcmd submodule update --init && ./scripts/remap.sh "$basedir" && ./scripts/decompile.sh "$basedir" && python3 $basedir/../disable_blacklisted.py $basedir/../blacklist.txt && ./scripts/init.sh "$basedir" && ./scripts/applyPatches.sh "$basedir" &&  python3 $basedir/../apply_mcpatches.py "$basedir/../patches/mcpatches" "$basedir") || (
     echo "Failed to build Paper"
     exit 1
 ) || exit 1
